config/overlays/e2e/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 namespace: olmv1-system
 
 resources:
-- ../../overlays/tls
+- ../../overlays/certs
```

Comment:
Are we only using it in our E2Es? Should we put it inside `config/overlays/e2e`?

Reply:
There are 3 different categories needed:
1.base
2. ext-dev-e2e
3. e2e

And they build upon each other (e2e adds coverage that isn't part of ext-dev-e2e). i could clean this up (I think there are now 4 categories), but I was aiming for minimal changes.

Reply:
@tmshort I currently only see only two overlays:
* tls (default)
* e2e (tls + coverage)

I think we should try to keep the number overlays to a minimum. If `overlays/certs` only introduced for the purpose of e2e tests - then IMO we should include this stuff into e2e.

> e2e adds coverage that isn't part of ext-dev-e2e

Gathering coverage from ext-dev-e2e will be beneficial. If absence of coverage in ext-dev-e2e makes it harder to add certs into existing setup - let's unify ext-dev-e2e and e2e (in a separate PR ideally) and then come back to this.

What do you think?

Reply:
I'm reducing the number of overlays. Anything beyond that is definitely new PR territory.